usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -226,3 +280,122 @@ def populate_response(self, results: list, has_next: bool) -> dict:
             "page_metadata": {"page": self.pagination["page"], "hasNext": has_next},
             "messages": [get_time_period_message()],
         }
+
+    def populate_elastic_response(
```

Comment:
What's the difference between the roles of `populate_elastic_response` and `construct_es_response`?

Reply:
well they do totally different things, but `populate_elastic_response` is pretty pointless so i will remove it